### PR TITLE
feat(#3756): Use a Single Format for All Error Messages

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Created one more class `LocationMessage` that is used as a default error message:
```
[2:0] error: 'Invalid program declaration'
```
It simply adds line and column where an error occured.

___
PS. Now we have two classes `UnderlinedMessage` and `LocationMessage`. They both have the same structure and similar methods. So, we can create an interface for them.
Moreover, we can change their names to `MsgUnderlined` and `MsgLocation`. I haven't done it in this PR in order to keep its size in some borders.
But we can change it during the PR review.

Closes: #3756.

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you review these changes, please?

File: eo-parser/src/main/java/org/eolang/parser/LocationMessage.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.parser;
+
+/**
+ * Error message that includes the location of the error.
+ * @since 0.50
+ */
+final class LocationMessage {
```
Comment (yegor256) [inline eo-parser/src/main/java/org/eolang/parser/LocationMessage.java:30]: @volodya-lombrozo maybe `LocatedMessage`? In general, it's not a good idea to have two glued together nouns as a name of a class. It's unclear what is it, a "location" or a "message"? Also, maybe it's better to name them `MsgLocated` and `MsgUnderlined`?

**Review by yegor256 — CHANGES_REQUESTED** [review]
@volodya-lombrozo looks good, just one comment above

**Comment by volodya-lombrozo** [comment]
@yegor256 Renamed classes according with your suggestions: https://github.com/objectionary/eo/pull/3762/commits/829f33494917cabc0337112efdb342692c8f299f
Can you have a look one more time, please?

**Review by yegor256 — APPROVED** [review]
LGTM

**Comment by yegor256** [comment]
@volodya-lombrozo thanks!

**Comment by 0crat** [comment]
@yegor256 Thanks for the review! You've earned +4 points for this: +4 as a basis; -10 for very few (4) comments; +10 to give you at least something.  Your running balance is [-16](https://www.eolang.org/zerocracy/objectionary-vitals.html).

**Pull request merged**